docs/content/docs/projects/CiSteps.md

```diff
@@ -406,7 +406,8 @@ configuration {}
 
 ### Using impact analysis in step
 
-Step could use [Impact analysis]({{< ref "/docs/ci/ImpactAnalysis.md" >}}). It is disabled by default.
+Step can use [Impact analysis]({{< ref "/docs/ci/ImpactAnalysis.md" >}}). 
```

Comment:
could -> can
We are talking about a general statement, not about something possible.